spec/image_optim_spec.rb

```diff
@@ -89,6 +89,41 @@ def temp_copy(image)
           end
         end
       end
+
+      describe 'skipping larger' do
```

Comment:
To clarify goal and reduce duplication it is better to extract common things using `before` and `let`